src/main/java/com/shippingcostestimator/enterprise/dao/ShipmentDAO.java

```diff
@@ -44,7 +44,6 @@ public Shipment saveEstimate(Shipment shipment) {
      */
     @Override
     public List<Shipment> fetchAllShipments() {
-        List<Shipment> fetchShipments = new ArrayList(allShipments.values());
-        return fetchShipments;
+        return (List<Shipment>) new ArrayList(allShipments.values());
```

Comment:
Removed a redundant variable to improve tech debt

Reply:
1) I'd keep it as it were.  If you try to cram too much into one line, and then you get a NullPointerException on that line (or some other exception), it can be very hard to track down.  There are several operations occurring on this one line.
2) You should not need to cast.  Avoid casting at all costs.
3) The ArrayList may require a generic identifier... hence the need to cast.